ansible/playbooks/tasks/cluster-bootstrap.yaml

```diff
@@ -233,6 +233,14 @@
       params tag={{ aws_stonith_tag}} pcmk_delay_max=15
   when: cloud_platform_is_aws
 
+- name: Set primary and secondary hostnames
+  ansible.builtin.set_fact:
+    primary_hostname: "{{ groups['hana'][0] }}"
```

Comment:
We use similar query to calculate `is_primary`. Differences are:
- all of them calculate it ad playbook level and not at task level
- other places are using `group.hana[0]`
- other places are using a combination like:
```
hosts: hana
vars:
        is_primary: "{{ ansible_play_hosts[0] == inventory_hostname }}"
```


Let's keep it like this here but revisit it everywhere in all files.